Update CONTRIBUTING.md (#19650)
cloudflare/cloudflare-docs: CONTRIBUTING.md

## Wrangler config

> If you're adding a code snippet to the docs that represents a Wrangler config file (`wrangler.toml`, `wrangler.json`, or `wrangler.jsonc`) make sure you wrap it with the `<WranglerConfig>` component, which ensures it’s rendered as both JSON and TOML e.g.

Reviewer: ```suggestion
If you are adding a code snippet to the docs that represents a Wrangler config file (`wrangler.toml`, `wrangler.json`, or `wrangler.jsonc`) make sure you wrap it with the `<WranglerConfig>` component, which ensures it is rendered as both JSON and TOML. For example:
```
